samples/tsv.js

```diff
@@ -1,11 +1,22 @@
+const parse = require('..');
+const assert = require('assert')
 
-// The package "should" must be installed:   
-// `npm install should`
-
-parse = require('..');
-should = require('should');
+parse(
+`1	2	3
+a	b	c`, {
+  delimiter: '\t'
+}, function(err, data){
+  if(err) throw err;
+  assert.deepStrictEqual(data, [ [ '1', '2', '3' ], [ 'a', 'b', 'c' ] ]);
+});
 
-parse( "1	2	3\ra	b	c", {delimiter: '\t'}, function(err, data){
+parse(
+`"1"	"2"	"3"
+"a"	"b"	"c"`, {
+  delimiter: '\t',
+  quote: "\"",
+  columns: true
```

Comment:
I am ok and grateful of the refactoring but I wish to preserve the original options, without `quote` and `columns`.

Reply:
Okay, I went ahead and removed that test case and reverted my change to the formatting of the original test case.  For what it's worth, your original test case was still there, just above this particular one.  The original one didn't cover the `columns` use case, so I thought it was a variant worth contributing, but that's fine. I _think_ I've discovered a bug specifically pertaining to tab-delimited values, but I haven't shored up my bug report yet so I'll hold off bringing that up until later.